docs/content/preview/releases/release-notes/v2.15.md

```diff
@@ -28,24 +28,26 @@ For an RSS feed of the release notes for all release series, point your feed rea
 
 ### Downloads
 
-<a class="download-binary-link" href="https://downloads.yugabyte.com/releases/2.15.3.2/yugabyte-2.15.3.2-b1-darwin-x86_64.tar.gz">
-  <button>
-    <i class="fa-brands fa-apple"></i><span class="download-text">macOS</span>
-  </button>
-</a>
-&nbsp; &nbsp; &nbsp;
-<a class="download-binary-link" href="https://downloads.yugabyte.com/releases/2.15.3.2/yugabyte-2.15.3.2-b1-linux-x86_64.tar.gz">
-  <button>
-    <i class="fa-brands fa-linux"></i><span class="download-text">Linux x86</span>
-  </button>
-</a>
-&nbsp; &nbsp; &nbsp;
-<a class="download-binary-link" href="https://downloads.yugabyte.com/releases/2.15.3.2/yugabyte-2.15.3.2-b1-el8-aarch64.tar.gz">
-  <button>
-    <i class="fa-brands fa-linux"></i><span class="download-text">Linux ARM</span>
-  </button>
-</a>
-<br />
+<ul class="nav yb-pills">
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.15.3.2/yugabyte-2.15.3.2-b1-darwin-x86_64.tar.gz">
+      <i class="fa-brands fa-apple"></i>
```

Comment:
These logos aren't showing up in the preview build. (Also heads-up, I'm **adding a release in v2.14.md today**, so this PR will need an update for that new release entry.)

Reply:
@polarweasel Yes, In pills logos doesn't show to have more space for text.
https://deploy-preview-14979--infallible-bardeen-164bc9.netlify.app/preview/develop/build-apps/java/ysql-ebean/

I am meeting with Mariam today to discuss more about pills/cards.

Reply:
@polarweasel Logos/Icons added back.
- https://deploy-preview-14979--infallible-bardeen-164bc9.netlify.app/preview/develop/build-apps/java/ysql-ebean/
- https://deploy-preview-14979--infallible-bardeen-164bc9.netlify.app/preview/releases/release-notes/v2.15/